drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -730,14 +753,8 @@ static int i2c_it8xxx2_transfer(const struct device *dev, struct i2c_msg *msgs,
 		data->ridx = 0;
 		data->err = 0;
 		data->msgs = &(msgs[i]);
-		data->addr_16bit = addr;
+		data->addr_16bit = addr << 1;
```

Comment:
This change is not related to the i2c_reset issue. Can it be broken out into a PR for fixing the i2c 7 bit address.

Reply:
OK, it has be split another commit:
https://github.com/zephyrproject-rtos/zephyr/pull/34809/commits/c150ec4f73641e522e2ac18a1f9da4eb5dff1f9c